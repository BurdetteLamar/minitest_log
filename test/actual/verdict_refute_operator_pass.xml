<log>
  <summary_ verdicts='2' failures='0' errors='0'/>
  <section_ name='Test' method='verdict_refute_operator?' type='pass'>
    <section_ name='Args=[0, :==, 1, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_operator?' outcome='passed' id='0' message='Message 0'>
        <object_1_ class='Integer' value='0'/>
        <operator_ class='Symbol' value=':=='/>
        <object_2_ class='Integer' value='1'/>
      </verdict_>
    </section_>
    <section_ name='Args=[false, :==, true, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_operator?' outcome='passed' id='1' message='Message 1'>
        <object_1_ class='FalseClass' value='false'/>
        <operator_ class='Symbol' value=':=='/>
        <object_2_ class='TrueClass' value='true'/>
      </verdict_>
    </section_>
  </section_>
</log>
