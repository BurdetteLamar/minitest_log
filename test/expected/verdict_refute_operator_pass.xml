<log>
  <summary verdicts='3' failures='0' errors='0'/>
  <section name='Test' method='verdict_refute_operator?' type='pass'>
    <section name='Args=[0, :==, 1]'>
      <verdict method='verdict_refute_operator?' outcome='passed' id='0' message='Message 0'>
        <object_1>
          0
        </object_1>
        <operator>
          :==
        </operator>
        <object_2>
          1
        </object_2>
      </verdict>
    </section>
    <section name='Args=[false, :==, true]'>
      <verdict method='verdict_refute_operator?' outcome='passed' id='1' message='Message 1'>
        <object_1>
          false
        </object_1>
        <operator>
          :==
        </operator>
        <object_2>
          true
        </object_2>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
